data/noiseprofiles.json

```diff
@@ -6096,6 +6096,37 @@
+            {"name": "GR II iso 1250", "iso": 1250, "a": [4.85006168132026e-05, 2.78186687566552e-05, 0.000111346408169291], "b": [1.51723398922851e-07, 1.34818971087117e-07, 2.61610456955968e-07]},
+            {"name": "GR II iso 1600", "iso": 1600, "a": [6.40294769692917e-05, 3.66450445938138e-05, 0.000149239113174306], "b": [2.19828843029918e-07, 1.78946410623565e-07, 3.45568466104492e-07]},
+            {"name": "GR II iso 2000", "iso": 2000, "a": [7.90961863054042e-05, 4.59174369512199e-05, 0.000184298451143703], "b": [3.27633719919272e-07, 2.62866715220962e-07, 4.86669225618842e-07]},
+            {"name": "GR II iso 2500", "iso": 2500, "a": [9.88051781702215e-05, 5.73436761130793e-05, 0.00023060646744293], "b": [4.49190412922312e-07, 3.60855441744992e-07, 6.48866677728509e-07]},
+            {"name": "GR II iso 3200", "iso": 3200, "a": [0.000133399916127845, 7.7233361771955e-05, 0.000312374243149196], "b": [6.10124438634669e-07, 4.73385142626478e-07, 8.64574358341396e-07]},
+            {"name": "GR II iso 4000", "iso": 4000, "a": [0.000166520792620046, 9.67477197876121e-05, 0.000388010185032697], "b": [8.8288765002446e-07, 6.84104908888748e-07, 1.24856668434693e-06]},
+            {"name": "GR II iso 5000", "iso": 5000, "a": [0.000208601239083288, 0.000121926801351858, 0.00048665050900771], "b": [1.07557417797185e-06, 7.73816550167386e-07, 1.47561359190245e-06]},
+            {"name": "GR II iso 6400", "iso": 6400, "a": [0.000285931671171879, 0.000165024181534589, 0.000656532224670442], "b": [1.48549224366654e-06, 1.08425903887038e-06, 1.9717079576522e-06]},
+            {"name": "GR II iso 8000", "iso": 8000, "a": [0.000359074172528058, 0.000213464717818508, 0.000848951225126015], "b": [1.8325639930754e-06, 1.28614427118221e-06, 2.48434675093179e-06]},
+            {"name": "GR II iso 10000", "iso": 10000, "a": [0.000470754538385169, 0.000278490538041487, 0.00106726186680657], "b": [2.19023981442033e-06, 1.44319831387867e-06, 2.72566693533878e-06]},
+            {"name": "GR II iso 12800", "iso": 12800, "a": [0.000654284461393812, 0.000388331036020567, 0.00140388667581021], "b": [3.16081319143953e-06, 2.10091677669883e-06, 3.61077876740858e-06]},
+            {"name": "GR II iso 16000", "iso": 16000, "a": [0.000824991340066458, 0.000501822035505979, 0.00177609313840165], "b": [4.19575226573771e-06, 2.76312184961383e-06, 4.82022236844907e-06]},
+            {"name": "GR II iso 20000", "iso": 20000, "a": [0.00101678286005032, 0.000616390668488325, 0.00223308013397197], "b": [5.97549949265273e-06, 4.07637139131783e-06, 6.64485549471867e-06]},
+            {"name": "GR II iso 25600", "iso": 25600, "a": [0.00142158488506293, 0.000890620215779999, 0.00278443118012513], "b": [7.80690184138186e-06, 5.38540540423255e-06, 8.35365270915029e-06]}
+          ]
```

Comment:
please create a ticket on https://redmine.darktable.org/projects/darktable/issues/new and add dt-noiseprofile-*.tar.gz
of noise profile. This is necessary to check the profile.

Reply:
I will do this. Am currently working on a better finetuned profile.